student.rb

```diff
@@ -1,4 +1,9 @@
+require_relative 'cheerable'
+
 class Student
+
+  include Cheerable
+
   attr_reader :age, :phase
   attr_accessor :name
```

Comment:
The `Employee` class has all the attr's as well, and a very similar initialize.  How could we DRY up that repeated code?